server/src/infra/repositories/smart-info.repository.ts

```diff
@@ -44,7 +44,7 @@ export class SmartInfoRepository implements ISmartInfoRepository {
   @GenerateSql({
     params: [{ ownerId: DummyValue.UUID, embedding: Array.from({ length: 512 }, Math.random), numResults: 100 }],
```

Comment:
Same thing here. It's now an array of userIds instead of a single ownerId

Reply:
Thanks, missed that!